app/src/main/java/com/gathering/android/common/KeyValueStorage.kt

```diff
@@ -0,0 +1,21 @@
+package com.gathering.android.common
+
+import android.content.SharedPreferences
+import javax.inject.Inject
+
+class KeyValueStorage @Inject constructor(
+    private val sharedPreferences: SharedPreferences
+) {
+
+    fun saveData(key: String, value: String?) {
+        sharedPreferences.edit().putString(key, value ?: "").apply()
+    }
+
+    fun getData(key: String): String? {
```

Comment:
getData should be renamed to getString():String, in the future we will have getInt, getBoolean , etc.